internal/storage/v2/clickhouse/tracestore/dbmodel/dbmodel.go

```diff
@@ -37,6 +36,12 @@ type Span struct {
 	Events []Event
 	Links  []Link
 
+	BoolAttributes   []Attribute[bool]
```

Comment:
Since events and links also have attributes we should have a reusable struct to represent them.